packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraProperties.java

```diff
@@ -150,10 +150,32 @@ public interface CameraProperties {
    * android.hardware.camera2.CameraCharacteristics#SCALER_AVAILABLE_MAX_DIGITAL_ZOOM key.
    *
    * @return Float Maximum ratio between both active area width and crop region width, and active
-   *     area height and crop region height
+   *     area height and crop region height.
    */
   Float getScalerAvailableMaxDigitalZoom();
 
+  /**
+   * Returns the minimum ratio between the default camera zoom setting and all of the available zoom
```

Comment:
Nit: missing period here and in the `@return` section; same in the other new method.